packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.tsx

```diff
@@ -1,35 +1,34 @@
-import React, { CSSProperties, useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import classNames from 'classnames';
 import { useCombobox } from 'downshift';
 import ArrowIcon from '../../../public/images/pointed-down-arrow-icon.svg';
 import reactNodeToString from '../../../utils/reactNodeToString';
 import { TGenericSizes } from '../types';
 import { WalletText } from '../WalletText';
+import { WalletTextField } from '../WalletTextField';
+import { WalletTextFieldProps } from '../WalletTextField/WalletTextField';
 import './WalletDropdown.scss';
 
-type TProps = {
+type TProps = Pick<WalletTextFieldProps, 'id' | 'label' | 'value'> & {
     icon?: React.ReactNode;
-    label?: React.ReactNode;
     list: {
         text?: React.ReactNode;
         value?: string;
     }[];
     listHeight?: Extract<TGenericSizes, 'lg' | 'md' | 'sm'>;
-    maxWidth?: CSSProperties['maxWidth'];
     onSelect: (value: string) => void;
-    type?: 'comboBox' | 'prompt';
-    value: string | undefined;
+    variant?: 'comboBox' | 'prompt';
 };
 
 const WalletDropdown: React.FC<TProps> = ({
     icon = false,
+    id = 'dropdown-text',
```

Comment:
find a way to assign a unique id, if they dont assign id and there are many dropdowns, there will be accessibility problems

Reply:
Good catch from @yashim-deriv. Maybe `useId` hook will solve this problem

Reply:
Would making id a required prop and forcing the addition of id on usage of WalletDropdown to distinguish the different dropdowns be a sufficient solution @sergei-deriv @yashim-deriv